src/frontend-samples/screen-space-effects-sample/ScreenSpaceEffectsUI.tsx

```diff
@@ -73,24 +73,27 @@ export default class ScreenSpaceEffectsUI extends React.Component<UIProps, UISta
     });
   }
 
-  private readonly _onChangeLensAngle = (event: React.ChangeEvent<HTMLInputElement>) => {
-    const lensAngle = Number(event.target.value);
-    this.setState({ lensAngle });
+  private readonly _onUpdateLensAngle = (values: readonly number[]) => {
+    this.setState({ lensAngle: values[0] });
   }
 
   public componentDidUpdate(_prevProp: UIProps, prevState: UIState) {
     const viewport = this.state.viewport;
     if (!viewport)
       return;
 
+    // Was a new iModel opened?
+    const newViewport = this.state.viewport?.viewportId !== prevState.viewport?.viewportId;
+
     let changed = false;
     if (this.state.lensAngle !== prevState.lensAngle) {
       changed = true;
       viewport.turnCameraOn(Angle.createDegrees(this.state.lensAngle));
+      viewport.invalidateScene();
```

Comment:
This should be `invalidateRenderPlan`. Remind me to fix `turnCameraOn` to do that itself.